specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -29260,6 +29669,116 @@
         "modelAsString": true
       }
     },
+    "WorkflowEnvelope": {
+      "description": "Workflow properties definition.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource id."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the resource name."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the resource type."
+        },
+        "kind": {
+          "type": "string",
+          "description": "The resource kind."
+        },
+        "location": {
+          "type": "string",
+          "description": "The resource location."
```

Comment:
Is this a tracked resource ? Just wanted to confirm

Reply:
What do you mean by that?

Reply:
Bump, any clarification?

Reply:
A resource that can be PUT/DELETE-d and has tags and a location known by ARM?

Reply:
It looks like technically it has a location but it is not PUT and DELETE able there so not truly a tracked resource?
"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/workflows/{workflowName}": {